api/events.js

```diff
@@ -40,3 +36,15 @@ export const findByIds = (apiUrl, headers, fetch) => async (eventIds) => {
   const events = await Promise.all(mappedEvents);
   return events.filter((event) => !event.status);
 };
+
+export const getCalendarSummary = (apiUrl, headers, fetch) => async ({
+  id,
+  locale,
+  format = 'lg',
+}) => {
+  const url = `${apiUrl}/events/${id.toString()}/calsum?format=${format}&langCode=${locale}_BE`;
+  const res = await fetch(url, {
+    headers: headers(),
+  });
+  return res.text();
```

Comment:
Shouldn't this be JSON?

Reply:
No, this endpoint returns plain text for some reason apparently.